markdown/ml/ml7_training_testing/index.md

```diff
@@ -18,13 +18,38 @@ outcomes: false
 # youtube:
 #   - id: XYZ (ID)
 #   - id: XYZ (ID)
-sketch: true
+sketch: false
 ---
 
 
-<div class="sketch" style="text-align:center;">
-Inhalt befindet sich im Aufbau<br>
-und wird rechtzeitig bereitgetellt.
-</div>
+## Kurze Übersicht
+
+### Training und Testing
+*	Der tatsächliche **Erfolg** eines Modells wird nicht durch niedrige Trainingskosten gemessen, sondern durch geringe Kosten auf ungesehenen Daten, i.e. **hohe Vorhersagekraft, gute Generalisierung**!
+
+*   Die Menge aller gelabelten Daten  in **Trainingsset und Testset** aufteilen, Testset nicht während des Trainings einsetzen!. 
+    *	$E_{in}$ bezeichnet den Fehler auf dem Trainingsset, auch **in-sample error**.
+    *	$E_{out}$ bezeichnet den Fehler auf dem gesamten Eingaberaum $X$, auch **out-of-sample error**. $E_{out}$ ist der eigentliche Indikator für den zukünftigen Erfolg des Modells, ist uns aber nicht zugänglich.
+    *	$E_{test}$ bezeichnet den Fehler auf dem Testset und ist eine **Näherung** für $E_{out}$.  
+
+    > Analogie:\
+    $E_{in}$ : Erfolg in Übungsaufgaben und Probeprüfungen.\
+    $E_{test}$ : Erfolg in Endprüfung. 
+
+*	Die Näherung $E_{test}$ sollte möglichst genau sein, damit es als ein verlässliches **Gütesiegel** dienen kann.
+	*	Das Testset sollte genug Daten enthalten. Üblicher Anteil an Testdaten:
+		*	bei  $|D| \approx 100.000 \rightarrow$  ca. 20%
+		*	bei  $|D| \approx 10.000.000 \rightarrow$  ca. 1%
+	    *	Beispiel: Hat man 1000 Beispiele im Testset, wird $E_{test}$ mit $\ge 98\%$ Wahrscheinlichkeit in der $\pm 5\%$ Umgebung von $E_{out}$ liegen (für theoretische Grundlagen und Herleitung siehe [@AbuMostafa2012, S. 39-69]).
+	*	Trainingsdaten und Testdaten sollten möglichst aus derselben Verteilung kommen, wie die zukünftigen **Real-World-Daten**.
+
+
+*	**Wichtige Bemerkung**: 
+    *   Testdaten nicht anfassen, bis das Modell Einsatzbereit ist!
+    *   Die Testdaten dürfen in **keinster Weise** bei der Auswahl der endgültigen Hyphotese eingesetzt werden, weder bei der Berechnung der Parameter (Training), noch bei der Bestimmung der Hyperparameter (Hyperparameter-Tuning). 
+    *   Sobald der Testfehler die Auswahl der endgültigen Hypothese beeinflusst, kann sie nicht mehr als "Gütesiegel" eingesetzt werden.\
```

Comment:
```suggestion
    *   Sobald der Testfehler die Auswahl der endgültigen Hypothese beeinflusst, kann sie nicht mehr als "Gütesiegel" eingesetzt werden.
```

Reply:
Auch hier bin ich nicht sicher, ob das mit dem `\` am Zeilenende in der Kombination mit Pandoc+Hugo so klappt wie gewünscht. Ein Umbruch und die passende Einrückung der nächsten Zeile sollten aber klappen :)